scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Hardcoded Database Credentials

This line contains hardcoded database credentials, which is a severe security risk. Database connection strings with credentials should never be stored directly in the source code[^1].

Recommendation: Remove these hardcoded credentials and use environment variables or a secure secret management system to store and retrieve sensitive information.

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html
```suggestion
const dbConnection = `postgresql://${process.env.DB_USER}:${process.env.DB_PASSWORD}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_NAME}` // Retrieve database credentials from environment variables
```